src/App.js

```diff
@@ -1,10 +1,27 @@
-import logo from './logo.svg';
+import { Routes, Route } from "react-router-dom";
 import './App.css';
 
+import {Home} from "./pages/home/home.jsx";
+import {Cart} from "./pages/cart/cart.jsx";
+import {WishList} from "./pages/wish-list/wish-list.jsx";
+import {ProductListing} from "./pages/product-listing/product-listing.jsx";
+import {Nav} from "./components/nav.jsx";
+import {Login} from "./pages/auth/login.jsx";
+import {SignUp} from "./pages/auth/sign-up.jsx"
+
```

Comment:
you can create an index file and export all components there. and while importing, can use single line import.

Reply:
sorry koushith, I gave direct product_list netlify route so, it's broken I think so. check out this: https://rad-longma-bee0dc.netlify.app/ it's working

Reply:
> you can create an index file and export all components there. and while importing, can use single line import.

yeah will refactor code